tests/test_fields.py

```diff
@@ -17,6 +18,12 @@
 )
 
 
+def field_names(model):
+    """"Return field names for `model`."""
+    fields = model._meta.get_fields()
+    return {field.name for field in fields}
```

Comment:
If you prefer the exclude approach, it's simply `find_packages(exclude=['tests'])`